### ISSUE TITLE
redundant directoryListing method in twisted.web2.static.File class

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2550]
|<img alt="stan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| stan reported|
|-|-|
|Trac ID|trac#2550|
|Type|defect|
|Created|2007-04-01 14:15:29Z|

The directoryListing method at line 287 doesn't appear to be used - render appears to do the equivalent itself.

<details><summary>Searchable metadata</summary>

```
trac-id__2550 2550
type__defect defect
reporter__stan stan
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__web2 web2
keywords__ 
time__1175436929000000 1175436929000000
changetime__1297615189000000 1297615189000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

Not going to fix this in web2 (See [#4821](https://github.com/twisted/twisted/issues/4821)).  `twisted.web.static.File` has a similar `directoryListing` method, and its `getChild` method does call it.  So no need to retarget this at the web component.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
